modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts

```diff
@@ -132,6 +143,23 @@ export class JournalEditArticlePage {
 		await this.journalPage.articleContentTextBox.press('Backspace');
 	}
 
+	async fillFriendlyURL(friendlyURL: string) {
+		if (await this.friendlyURLInput.isHidden()) {
+			await this.friendlyUrlToggle.click();
+		}
+		await this.friendlyURLInput.fill(friendlyURL);
+	}
+
+	async createBasicArticleWithFriendlyURL(site, page) {
+		await this.journalPage.goto(site.friendlyUrlPath);
+		await this.journalPage.goToCreateArticle('Contenu web basique');
```

Comment:
This is valid only for french. We could change the function name, or better, pass an optional parameter to the function, something like: 

```
async createBasicArticleWithFriendlyURL(site: string, page: string, articleTitle?: string) {
....
     await this.journalPage.goToCreateArticle(articleTitle || 'Basic Web Content');
....
}
```

and call it from the test 
```
await journalEditArticlePage.createBasicArticleWithFriendlyURL(
	site,
	page,
       'Contenu web basique'
);
```

Reply:
I like this: articleTitle || 'Basic Web Content :)

Reply:
also needed to change New button for something not language dependant. page.locator('button[data-qa-id="creationMenuNewButton"].d-md-flex.d-none');

Reply:
Ok, are those clases `.d-md-flex.d-none` necessary? Lets see if it passes CI and we can forward it